deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet

```diff
@@ -6,7 +6,7 @@
   sumologicCollector:: {
     remoteWriteConfigs+: [
       {
-        url: $._config.sumologicCollectorSvc + "prometheus.metrics.state",
```

Comment:
Are those changes OK? Why they were expanded now with this PR?

Reply:
This has been fixed, by merging master into the branch.